src/LAMMPS_oxDNA.py

```diff
@@ -80,5 +80,90 @@ def quat_to_exyz(myquat):
     configuration_file = basename + ".oxdna"
     system.print_lorenzo_output(configuration_file, topology_file)
 
+    if len(sys.argv) == 3:
+
+        f = open(sys.argv[2],'r')
+        cf = open(configuration_file,'w')
```

Comment:
Could you turn these two lines into something a bit more "pythonic" such as `with open(sys.argv[2],'r') as f, open(configuration_file,'w') as cf:` and maybe change the name of `f` and `cf` into something a bit more explicit?

Reply:
No problem.

Reply:
Thanks! Note that with `with` you don't need to explicitly `close` the files!